lib/server-utils.ts

```diff
@@ -242,11 +243,11 @@ export function getDataForStaticFile(htmlReporter: HtmlReporter, pluginConfig: R
     };
 }
 
-export async function initializeCustomGui(hermione: Hermione, {customGui}: ReporterConfig): Promise<void> {
+export async function initializeCustomGui(testplane: Testplane, {customGui}: ReporterConfig): Promise<void> {
     await Promise.all(
         _(customGui)
             .flatMap<CustomGuiItem>(_.identity)
-            .map((ctx) => ctx.initialize?.({hermione, ctx}))
+            .map((ctx) => ctx.initialize?.({testplane, hermione: testplane, ctx}))
```

Comment:
`testplane` and `hermione` props are `Object.is` copies for backward compatibility